elisp/intero.el

```diff
@@ -893,7 +910,7 @@ Other arguments are IGNORED."
                 (funcall cont
                          (cl-remove-if-not
                           (lambda (candidate)
-                            (string-match (concat "^" prefix) candidate))
+                            (string-match-p (concat "^" prefix) candidate))
```

Comment:
If `prefix` isn't a regexp, which it shouldn't be, this could/should simply be `string-prefix-p`.